dronecan/sensors/magnetometer/1043.MagneticFieldStrengthHiRes.uavcan

```diff
@@ -0,0 +1,5 @@
+#
+# Magnetic field readings, in Gauss, in body frame, high resolution version
+
```

Comment:
Timestamp could actually start to be helpful here (obviously depending on what you have and what you're doing with it).

Reply:
i don't think the delay from the capture will be significant, and we already capture the time the first packet on the CAN bus arrives
this is tested and deployed now by a user, so I will merge